homeassistant/components/sensor/currencylayer.py

```diff
@@ -69,6 +69,11 @@ def __init__(self, rest, base, quote):
         self._state = None
 
     @property
+    def unit_of_measurement(self):
+        """Return the unit of measurement of this entity, if any."""
+        return self._quote
+
+    @property
     def name(self):
         """Return the name of the sensor."""
         return '{} {}'.format(self._base, self._quote)
```

Comment:
`self._quote` looks now like a duplicate.

Reply:
I followed your approach in [fixer.io](https://github.com/home-assistant/home-assistant/blob/dev/homeassistant/components/sensor/fixer.py#L78). Basically, `quote` currency is the unit of measurement.

Reply:
Sorry, I meant adding `quote` to the name. Having it as unit of measurement makes more sense.